account_invoice_change_currency/models/account_change_currency.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2017-2018 Vauxoo
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+    original_price_unit = fields.Monetary(
+        default=lambda self: self.price_unit,
```

Comment:
Defaults are run over an empty recordset, because they're run when the record doesn't exist already. So trying to access any field will retrieve always falsy values, in this case 0.0

Reply:
Done!